.github/workflows/ssh-runner.yml

```diff
@@ -53,11 +53,21 @@ jobs:
       - name: NVIDIA-SMI
         run: |
           nvidia-smi
-      
+
+      - name: Store Slack infos
+        #because the SSH can be enabled dynamically if the workflow failed, so we need to store slack infos to be able to retrieve them during the waitforssh step
+        shell: bash
+        run: |
+          if [ "${{ secrets[format('{0}_{1}', github.actor, 'SLACK_ID')] }}" != "" ]; then
```

Comment:
If a GitHub secret is added to the repository: for example, `YDSHIEH_SLACK_ID` with a value being the corresponding slack user ID

Reply:
Quick Q, should the slack id not be an argument to  workflow_dispatch 🤗

Reply:
It is an ID like `X13JMGOCTSU`, and has to be copied from the Slack user profile.
Being an input might add some friction to this process (but browser usually remember input)

The IMPORTANT is: we want to keep it as a secret (and not show on log), like what we did so for 

> ${{ secrets.SLACK_CIFEEDBACK_CHANNEL }}